lib/experimental/interactivity-api/scripts.php

```diff
@@ -7,20 +7,21 @@
  */
 
 /**
- * Move interactive scripts to the footer. This is a temporary measure to make
- * it work with `wp_store` and it should be replaced with deferred scripts or
- * modules.
+ * Move interactive scripts to the footer and make them load with the defer strategy.
+ * This ensures they work with `wp_store`.
  */
 function gutenberg_interactivity_move_interactive_scripts_to_the_footer() {
 	// Move the @wordpress/interactivity package to the footer.
 	wp_script_add_data( 'wp-interactivity', 'group', 1 );
+	wp_script_add_data( 'wp-interactivity', 'strategy', 'defer' );
 
 	// Move all the view scripts of the interactive blocks to the footer.
 	$registered_blocks = \WP_Block_Type_Registry::get_instance()->get_all_registered();
 	foreach ( array_values( $registered_blocks ) as $block ) {
 		if ( isset( $block->supports['interactivity'] ) && $block->supports['interactivity'] ) {
 			foreach ( $block->view_script_handles as $handle ) {
 				wp_script_add_data( $handle, 'group', 1 );
+				wp_script_add_data( $handle, 'strategy', 'defer' );
```

Comment:
I guess that if we are making all frontend block scripts deferred by default, we don't need to do it for the `supports.interactivity` ones again. We still need to add `defer` to `wp-interactivity`, though.

Reply:
Cool. Done in aab0b4bc93a9742e2e3c43fc3556e966aa00241d